jekyll/_cci2/workflows.md

```diff
@@ -107,6 +107,38 @@ The dependencies are defined by setting the `requires:` key as shown. The `deplo
 
 See the [Sample Sequential Workflow config](https://github.com/CircleCI-Public/circleci-demo-workflows/blob/sequential-branch-filter/.circleci/config.yml) for a full example.
 
+## Job Contexts Example
+
+The following example shows a workflow with four sequential jobs that use shared environment variables. 
+
+The following `config.yml` snippet is an example of a sequntial job workflow configured to use the resources defined in the `org-global` context:
+
+```
```

Comment:
I'd suggest showing an example where not every job uses the context. Perhaps in this case just the test1 and test2 would be appropriate.